tests/testsuite.src/syn_definition.at

```diff
@@ -2878,3 +2878,76 @@ prog.cob:6: error: 'DEPTHDR' is not defined
 
 AT_CLEANUP
 
+
+# Note: the three tests below should be combined in a single one (when fully supported)
+AT_SETUP([POINTER TO typedef-name])
+AT_KEYWORDS([POINTER TYPEDEF])
+
+AT_DATA([prog.cob], [
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. prog.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 data-type PIC X(10) USAGE DISPLAY IS TYPEDEF.
+       01 data-ptr1 USAGE POINTER data-type.
+       01 data-ptr2 USAGE POINTER TO data-type.
+       PROCEDURE DIVISION.
+           STOP RUN.
+       END PROGRAM prog.
+])
+
+AT_CHECK([$COMPILE_ONLY -std=default -Wno-pending prog.cob], [0], [], [])
+
```

Comment:
drop the empty line here as wall as -std=default, only have the -Wno-pending if needed, same for the other tests